src/util.js

```diff
@@ -31,7 +68,7 @@ export var isEmpty = function (o) {
     return typeof o === 'undefined' || o === null || (isString(o) && o.length === 0) || (typeof o === 'object' && Object.keys(o).length === 0);
 };
 export var notEmpty = function (o) {
-    return !c3_chart_internal_fn.isEmpty(o);
+    return !c3.chart.internal.fn.isEmpty(o);
```

Comment:
You already have `c3.chart.internal.fn` defined in this module, so you don't need to import it's alias from `./core`